integrations/hibob/.port/resources/port-app-config.yml

```diff
@@ -0,0 +1,21 @@
+resources:
+  - kind: list
+    selector:
+      query: .name | <your-team-list-name>
```

Comment:
can you elaborate on what you are trying achieve here? Ideally I would like this to be auto discoverable on setup either from the hibob api or configured by the user installing it

Reply:
let me know if you would like to schedule a call to brainstorm